Examples/Tutorials/Tutorial-DpdkL2Fwd/Makefile

```diff
@@ -1,4 +1,8 @@
-include /usr/local/etc/PcapPlusPlus.mk
+PKGCONG := pkg-config
+
+PCAPPP_BUILD_FLAGS := "-fPIC"
+PCAPPP_INCLUDES := $(shell $(PKGCONF) --cflags PcapPlusPlus)
+PCAPPP_LIBS := $(shell $(PKGCONF) --libs PcapPlusPlus)
```

Comment:
Why do we need to keep the makefiles for the tutorials?

Reply:
We can remove them, it's just an example how to build with make and the CI validate that it works fine.